### PR TITLE
Folderify our gulp sass task

### PR DESCRIPTION
**Pull request by kenkouot**
This PR applies the same assets/folder strategy that we use for scripts, svg and templates to our scss/css files. As we laid the foundation for multi-tenancy in Mercury, @lizlux and I realized that this would make it easier to identify which assets were shared, and which were not.

For more information on gulp-folders, check out [hakubo/gulp-folders](https://github.com/hakubo/gulp-folders).

### CODE REVIEW

**Comment by lizlux** [comment]
LGTM

File: front/styles/auth/app.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+	background: salmon;
```
Comment (kenkouot) [inline front/styles/auth/app.scss]: This is just a seed file for @lizlux's login work. Salmon is also my favorite color code.

Comment (bkoval) [inline front/styles/auth/app.scss]: :dancers:

**Comment by hakubo** [comment]
could we just name it `main`?

**Comment by kenkouot** [comment]
Sure!

File: gulp/tasks/sass.js [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+gulp.task('sass', folders(paths.src, function (folder) {
 	return piper(
-		gulp.src(paths.src),
+		gulp.src([
+			'!' + path.join(paths.src, folder, paths.partials),
+			path.join(paths.src, folder, paths.compile)
+		]),
 		sass(options),
-		//currently support for map is broken
 		prefixer(['last 2 version', '> 1%', 'ie 8', 'ie 7'], {
```
Comment (lizlux) [inline gulp/tasks/sass.js]: Why do we have IE 7 and 8 in here? If anything, shouldn't we have IE 9?

File: gulp/tasks/sass.js [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
 	options = require('../options').sass,
+	path = require('path'),
 	paths = require('../paths').styles;
 
-gulp.task('sass', function () {
+gulp.task('sass', folders(paths.src, function (folder) {
 	return piper(
-		gulp.src(paths.src),
+		gulp.src([
+			'!' + path.join(paths.src, folder, paths.partials),
```
Comment (lizlux) [inline gulp/tasks/sass.js]: Just read [here](https://github.com/wearefractal/vinyl-fs#srcglobs-opt) that negations should follow positive globs. Looking at the output though, it didn't seem to make a difference for `ember-main.css` when I switched the order. Thoughts @kenkouot?

**Pull request merged**